class JoinInException(Exception):
    pass


class TransactionException(JoinInException):
    pass
